master/buildbot/process/buildstep.py

```diff
@@ -871,6 +872,9 @@ def startCommand(self, cmd, errorMessages=[]):
 
         # stdio is the first log
         self.stdio_log = stdio_log = self.addLog("stdio")
+        # enable timestamps if requested
+        self.stdio_log.setTimestampsMode(self.timestamp_stdio)
```

Comment:
Ok, what do you think about changing prepend_timestamps -> timestamp_stdio to keep the consistency of the name of the parameter?